script.js

```diff
@@ -16,82 +16,139 @@ function selectShows() {
-  makeSearch();
-  const dropdownEpisodeMenu = document.querySelector(".episode-select");
-  dropdownEpisodeMenu.style.display = "none";
 
+function titleCaseInsensitive(showA, showB) {
+  let nameA = showA.name.toLowerCase();
+  let nameB = showB.name.toLowerCase();
+  if (nameA < nameB) {
+    return -1;
+  }
+  if (nameA > nameB) {
+    return 1;
+  }
+  return 0;
+}
+
+
+function makeCards(all) {
   const wrapper = document.querySelector("#wrapper");
-  allShows.forEach(show => {
-    let {image, id, name, summary, rating, genres, status, runtime, url} = show;
+
+  all.forEach(card => {
+    let { image, id, name, summary, url, rating, genres, status, runtime, season, number } = card;
     const getImage = image !== null ? image.medium : "";
-    wrapper.innerHTML += `
-        <div class="col card-wrapper">
-          <div class="card bg-light p-3 h-100 pt-3">
-            <div class="card">
-              <h5 class="card-title title d-flex justify-content-center text-center" data-id="${id}">${name}</h5>
-            </div>
-            <img src="${getImage}" class="img mb-2 mt-2 px-3" alt="no image found" />
-            ${summary}
-            <div class="card px-2 pt-2 mb-2">
-              <p class="show-detail"><span class="show-detail-bold">Rated:</span> ${rating.average}</p>
-              <p class="show-detail"><span class="show-detail-bold">Genres:</span> ${genres}</p>
-              <p class="show-detail"><span class="show-detail-bold">Status:</span> ${status}</p>
-              <p class="show-detail"><span class="show-detail-bold">Runtime:</span> ${runtime}</p>
-            </div>
-            <a href="${url}">See Details</a>
-          </div>
-        </div>`
+
+    const episodeSeason = `${season > 9 ? season : "0" + season}`;
+    const episodeNumber = `${number > 9 ? number : "0" + number}`;
+    const episodeTitle = `${name} - S${episodeSeason}E${episodeNumber}`;
+    const title = season === undefined && number === undefined ? name : episodeTitle
+
+    const cardWrapper = document.createElement("div");
+    cardWrapper.setAttribute("class", "col card-wrapper");
+
+    const cardContainer = document.createElement("div");
+    cardContainer.setAttribute("class", "card card-container bg-light p-3 h-100 pt-3");
+
+    const cardTitleContainer = document.createElement("div");
+    cardTitleContainer.setAttribute("class", "card");
+
+    const cardTitle = document.createElement("h5");
+    cardTitle.setAttribute("class", "card-title title d-flex justify-content-center text-center data-id");
+    cardTitle.dataset.id = id;
+    cardTitle.innerText = title;
+    cardTitleContainer.appendChild(cardTitle);
+
+    const cardImage = document.createElement("img");
+    cardImage.src = getImage;
+    cardImage.setAttribute("class", "img mb-2 mt-2 px-3");
+
+    const cardSummary = document.createElement("div");
+    cardSummary.innerHTML = summary;
+
+    const showDetails = document.createElement("a");
+    showDetails.href = url;
```

Comment:
The 'see details' link goes to the Tv-maze websites page for the show. Maybe you could have it linked to the page you created for the show when clicked instead.